.github/DISCUSSION_TEMPLATE/component-proposals.yml

```diff
@@ -0,0 +1,93 @@
+title: "Component proposal: [Component name]"
+labels: ["Type: New component proposal"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for sharing your new component suggestion for the U.S. Web Design System (USWDS).
+        </p><p>
+        We ask for a lot of detail in this form, but it’s okay to start with basic information. Only the fields marked with an asterisk (*) are required at this time. You can work with the community to add more detail after you start this discussion. It helps if you link to supporting evidence and research wherever you can.
+        </p><p>
+        <a href="https://github.com/uswds/uswds/discussions/5764">Learn more about the component proposal process</a>.
```

Comment:
@sarah-sch and @finekatie 

Would either of you be able to review and edit this intro text? I mostly pulled this copy from the "What your discussion needs" section in the [pinned post](https://github.com/uswds/uswds/discussions/5764), but made some edits to fit the format.